### PR TITLE
Cloud Functions samples.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: functions/helloworld/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Google Cloud Functions Hello World! sample
+
+This sample shows a single exported function–a basic hello world example.
```
Comment (jonparrott) [inline functions/helloworld/README.md]: Nit: spaces around the emdash? Or better, a colon?

File: functions/helloworld/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+gcloud alpha functions deploy helloworld --bucket <your-bucket-name> --trigger-http
+```
+
+## Test the function
+
+```
+gcloud alpha functions call helloworld
+```
+
+Running the above command should print "Hello World!".
```
Comment (jonparrott) [inline functions/helloworld/README.md]: Can you show them how to trigger it via curl?

File: test/functions/helloworld.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var assert = require('assert');
+
+var helloworldSample = require('../../functions/helloworld');
+
+describe('functions/helloworld', function () {
```
Comment (jonparrott) [inline test/functions/helloworld.test.js]: Sigh. More code to test than to write. I generally abhor these kinds of tests as its usually more of a burden to maintain the tests than the code, but it's fine. I'm just venting.

**Comment by jonparrott** [comment]
LGTM with small nits.

File: functions/helloworld/README.md (hunk 2 above)
Comment (jmdobry) [inline functions/helloworld/README.md]: Fixed

File: functions/helloworld/README.md (hunk 1 above)
Comment (jmdobry) [inline functions/helloworld/README.md]: Fixed

**Pull request merged**